chia/protocols/farmer_protocol.py

```diff
@@ -42,6 +66,7 @@ class DeclareProofOfSpace(Streamable):
     farmer_puzzle_hash: bytes32
     pool_target: Optional[PoolTarget]
     pool_signature: Optional[G2Element]
+    include_signature_source_data: Optional[bool]
```

Comment:
I don't think this bool needs to be optional. Presumably `None and `False` are interpreted  the same.